plugin/db/pg/pg.go

```diff
@@ -135,22 +138,17 @@ func guessDSN(username, password, hostname, port, database, sslCA, sslCert, sslK
 	}
 	dsn := strings.Join(tokens, " ")
 
-	var guesses []string
 	if database != "" {
```

Comment:
This changes the logic. Before if the user specifies the database name, we are able to return error if connection info is incorrect.

If we want to save an extra Ping, then we'd better add a comment

Reply:
Done.